sdk/storage/azure-storage-file-share/README.md

```diff
@@ -415,7 +415,7 @@ Taking the fileClient in KeyConcept, [`${fileClient}`](#file) with data of "defa
 ```java readme-sample-uploadDataToStorageBiggerThan4MB
 byte[] data = "Hello, data sample!".getBytes(StandardCharsets.UTF_8);
 
-long chunkSize = ShareFileAsyncClient.FILE_DEFAULT_BLOCK_SIZE;
+long chunkSize = 4 * 1024 * 1024L;
```

Comment:
Was this mistakenly generated?

Reply:
Nope, looks like this is what the sample code uses now

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/src/samples/java/com/azure/storage/file/share/ReadmeSamples.java#L242